lib/utils.js

```diff
@@ -71,25 +71,22 @@ function normalizeBinary (binaryPath, platform, arch) {
     // this is used when reading the registry goes wrong.
     function fallBack () {
       var programFilesVar = "ProgramFiles";
-      if (arch === "(64)") {
-        programFilesVar = "ProgramFiles(x86)";
-      }
+     // if (arch === "(64)") {
+     //  programFilesVar = "ProgramFiles(x86)";
+     //  console.log("Please use firefox x64 for Windows x64");
+     // }
       resolve(path.join(process.env[programFilesVar], appName, "firefox.exe"));
     }
 
-    var rootKey = '\\Software\\Mozilla\\';
-    if (arch === "(64)") {
-      rootKey = '\\Software\\Wow6432Node\\Mozilla';
-    }
-    rootKey = path.join(rootKey, appName);
-
+    var rootKey = path.join('\\Software\\Mozilla\\', appName);
+ 
     return when.promise(function(resolve, reject) {
       var versionKey = new Winreg({
         hive: Winreg.HKLM,
         key: rootKey
       });
       versionKey.get("CurrentVersion", function(err, key) {
-        return (err) ? reject() : resolve(key.value);
+        return (err) ? console.log("Please use firefox x64 for Windows x64") : resolve(key.value);
```

Comment:
this would need to be return an error, more like:

    return (err) ? reject(new Error("only firefox x64 is supported on Windows x64")) : resolve(key.value);